javascript/jQuerry.js

```diff
@@ -0,0 +1,122 @@
+
+var stack = [];
+
+function isNumeric(num){
```

Comment:
Always leave en empty space between `)` and `{`.
Also, there is already a function exposed by jQuery that does this: `$.isNumeric(...)`.